src/plugins/objectMigration/plugin.js

```diff
@@ -36,8 +36,8 @@ export default function () {
         }
 
         let wrappedFunction = openmct.objects.get;
-        openmct.objects.get = function migrate(identifier) {
-            return wrappedFunction.apply(openmct.objects, [identifier])
+        openmct.objects.get = function migrate() {
+            return wrappedFunction.apply(openmct.objects, [...arguments])
```

Comment:
How can we test this change? Would this affect all object migrations?

Reply:
Not entirely sure how we'd test it. But this should actually make things a bit better. It now will include abort signals, which it previously wasn't. What I'd really like, is for this to not patch the `get` method and just make these migrations `getInterceptors`.